### PR TITLE
Add missing meta descriptions

### PR DESCRIPTION
**Pull request by walsehgal**
Add missing meta description in docs.

### CODE REVIEW

**Comment by walsehgal** [comment]
@traya1

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: migration/Box-to-OneDrive-and-SharePoint-migration-guide.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ ms.collection:
 search.appverid: MET150
 ms.custom: 
 ms.assetid: 
-
+description: "Learn how to evaluate the environment, prepare and migrate content from Box to OneDrive in Office 365."
```
Comment (JohanFreelancer9) [inline migration/Box-to-OneDrive-and-SharePoint-migration-guide.md]:
```suggestion
description: "Learn how to evaluate the environment, prepare, and migrate content from Box to OneDrive in Office 365."
```

File: migration/Fileshare-to-ODSP-migration-guide.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ ms.collection:
 search.appverid: MET150
 ms.custom: 
 ms.assetid: 
-
+description: "Learn how to evaluate the environment, prepare and migrate content from File sharing systems to OneDrive and SharePoint in Office 365."
```
Comment (JohanFreelancer9) [inline migration/Fileshare-to-ODSP-migration-guide.md]:
```suggestion
description: "Learn how to evaluate the environment, prepare, and migrate content from File sharing systems to OneDrive and SharePoint in Office 365."
```

File: migration/MySites-to-OneDrive-Migration-Guide.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ ms.collection:
 - SPMigration
 - M365-collaboration
 search.appverid: MET150
-
+description: "Learn how to evaluate the environment, prepare and migrate content from MySites to OneDrive in Office 365."
```
Comment (JohanFreelancer9) [inline migration/MySites-to-OneDrive-Migration-Guide.md]:
```suggestion
description: "Learn how to evaluate the environment, prepare, and migrate content from MySites to OneDrive in Office 365."
```

File: migration/using-the-sharepoint-migration-tool-reports.md [hunk 4]
```diff
@@ -14,6 +14,7 @@ mscollection:
 - SPMigration
 - M365-collaboration
 search.appverid: MET150
+description: "Learn how to assess reports generated by the SharePoint Migration Tool to manage, audit and troubleshoot your migration process."
```
Comment (JohanFreelancer9) [inline migration/using-the-sharepoint-migration-tool-reports.md]:
```suggestion
description: "Learn how to assess reports generated by the SharePoint Migration Tool to manage, audit, and troubleshoot your migration process."
```

**Comment by traya1** [comment]
Hi @walsehgal, 

Can you close this PR and open a new PR that points to the correct parent branch? Ping me if you need any help.

**Comment by walsehgal** [comment]
> 
> 
> Hi @walsehgal,
> 
> Can you close this PR and open a new PR that points to the correct parent branch? Ping me if you need any help.

Done!

**Pull request merged**